src/wallet/walletReducer.js

```diff
@@ -1,8 +1,19 @@
-import { OPEN_TRANSFER, CLOSE_TRANSFER } from './walletActions';
+import {
+  OPEN_TRANSFER,
+  CLOSE_TRANSFER,
+  GET_GLOBAL_PROPERTIES,
+  GET_USER_TRANSACTIONS,
+  GET_USER_EST_ACCOUNT_VALUE,
+} from './walletActions';
```

Comment:
We mostly use `import * as xTypes from './xActions';` and use `xTypes.ACTION_TYPE` inside reducer. I think we should stick to this naming convention.